packages/ui/src/components/va-pagination/VaPagination.demo.vue

```diff
@@ -1,216 +1,376 @@
 <template>
-  <VbDemo>
+  <VbDemo class="va-pagination-demo">
     <VbCard>
       Current Page
       <input
-        type="number"
         v-model.number="activePage"
+        aria-hidden="true"
+        type="number"
       />
     </VbCard>
     <VbCard>
       <table class="table table-bordered">
         <tr>
           <th>Description</th>
-          <th>Va Pagination</th>
+          <th>VaPagination</th>
         </tr>
         <tr>
-          <td>Default Pagination</td>
+          <td>Default</td>
           <td>
             <VbCard>
               <va-pagination
+                v-model="activePage"
                 :pages="5"
+              />
+            </VbCard>
+          </td>
+        </tr>
+        <tr>
+          <td>Presets</td>
+          <td>
+            <VbCard>
+              <va-pagination
                 v-model="activePage"
+                :pages="5"
+                preset="default"
+              />
+              <va-pagination
+                v-model="activePage"
+                :pages="5"
+                preset="primary"
+              />
+              <va-pagination
+                v-model="activePage"
+                :pages="5"
+                preset="secondary"
               />
             </VbCard>
           </td>
         </tr>
         <tr>
-          <td>Pagination With Visible Pages</td>
+          <td>Gapped</td>
           <td>
             <VbCard>
               <va-pagination
+                v-model="activePage"
+                :pages="5"
+                gapped
+                preset="default"
+              />
+              <va-pagination
+                v-model="activePage"
+                :pages="5"
+                gapped
+                preset="primary"
+              />
+              <va-pagination
+                v-model="activePage"
+                :pages="5"
+                gapped
+                preset="secondary"
+              />
+            </VbCard>
+          </td>
+
+        </tr>
+        <tr>
+          <td>Visible Pages</td>
+          <td>
+            <VbCard>
+              <va-pagination
+                v-model="activePage"
                 :pages="10"
                 :visible-pages="3"
+                preset="default"
+              />
+              <va-pagination
                 v-model="activePage"
+                :pages="10"
+                :visible-pages="3"
+                preset="primary"
+              />
+              <va-pagination
+                v-model="activePage"
+                :pages="10"
+                :visible-pages="3"
+                preset="secondary"
               />
             </VbCard>
           </td>
         </tr>
         <tr>
-          <td>Pagination Sizes</td>
+          <td>Bordered</td>
           <td>
             <VbCard>
               <va-pagination
```

Comment:
<img width="209" alt="image" src="https://user-images.githubusercontent.com/5394573/189232370-2e51e40c-bfaf-4a3f-a2c0-b61b5625e43c.png">

Left and right buttons stop working if modelValue is out of bounds (i.e. for pages="5", :modelValue="20")